### ISSUE TITLE
[Adaptive Card iOS - Image.BackgroundColor.json]: Voice over does not announce the the descriptive information about the image background color.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7345]
https://microsoft.visualstudio.com/OS/_workitems/edit/37841998
